.gitignore

```diff
@@ -9,7 +9,6 @@ build.properties
 salesforce.schema
 Referenced Packages
 /.sfdx
-**/.vscode/
```

Comment:
Now that we've merged the .gitignore pull request into master, we don't want to revert it...

Reply:
@robertbwatson ugh having a really hard time with multiple branch commits across two repos.... just undid that change that probably happened after I resolved merge conflict